test/integration/integration-entrypoint.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+#
+# Sifnode entrypoint.
+#
+
+#
+# Daemon.
+#
+start_daemon() {
+  sifnoded start --rpc.laddr tcp://0.0.0.0:26657
+}
+
+#
+# Rest server.
+#
+start_rest_server() {
+  sifnodecli rest-server --laddr tcp://0.0.0.0:1317 &
+}
+
+#
+# Start relayer.
+#
+start_relayer() {
+  wait_for_rpc
+  ebrelayer init tcp://0.0.0.0:26657 "$ETHEREUM_WEBSOCKET_ADDRESS" \
+                                             "$ETHEREUM_CONTRACT_ADDRESS" \
+                                             "$MONIKER" \
+                                             "$MNEMONIC" \
+                                             --chain-id "$CHAINNET" \
+                                             --gas 200000 \
```

Comment:
Maybe change the gas to 300000. We had some issues at 200000.

Reply:
Is this the other way around?  Shouldn't we use the amount that's been a problem in the past?

Reply:
Talked about this in the meeting and we changed it to 300000.